--- dfu-programmer-0.5.4/configure.ac.orig	2011-01-17 07:58:54.000000000 +0100
+++ dfu-programmer-0.5.4/configure.ac	2012-11-04 09:04:29.028027559 +0100
@@ -27,7 +27,7 @@
   PKG_CHECK_MODULES(LIBUSB_1_0, [ libusb-1.0 >= 1.0.0 ], have_libusb_1_0=yes, have_libusb_1_0=no)
   if test "$have_libusb_1_0" = "yes"; then
     CFLAGS="$CFLAGS $LIBUSB_1_0_CFLAGS"
-    LDFLAGS="$LDFLAGS $LIBUSB_1_0_LIBS"
+    LIBS="$LIBS $LIBUSB_1_0_LIBS"
     HAVE_USB=yes
   fi
 fi
